longevity_test.py

```diff
@@ -179,7 +179,16 @@ def test_custom_time(self):
         if alternator_port:
             endpoint_url = 'http://{}:{}'.format(normalize_ipv6_url(self.db_cluster.nodes[0].external_address),
                                                  alternator_port)
-            alternator_create_table(endpoint_url, test_params=self.params)
+
+            if self.params.get('alternator_enforce_authorization'):
+                with self.cql_connection_patient(self.db_cluster.nodes[0]) as session:
+                    session.execute("""
+                        INSERT INTO system_auth.roles (role, salted_hash) VALUES (%s, %s)
+                    """, (self.params.get('alternator_access_key_id'),
+                          self.params.get('alternator_secret_access_key')))
+
+            dynamodb_primarykey_type = self.params.get('dynamodb_primarykey_type', default='HASH')
```

Comment:
why is there a default here and not in defaults yaml?

Reply:
I was here cause of some unit-tests, but it can be removed

Reply:
fixed